Module & type warning for video file

Vscode is showing an error for the video file import. This is after the tsconfig has been updated, and the file does exist. However the source file won't always exist. Should I (and the readme) be explicitly typing this import?

<img width="751" alt="image" src="https://github.com/muxinc/next-video/assets/166/e776d694-793c-4075-bbea-0f1b4fe66e53">

Then also this warning. Same issue or different?
<img width="1149" alt="image 2" src="https://github.com/muxinc/next-video/assets/166/404ea4cc-a8f9-47bf-9442-7339c55a0158">

got this TS error too when running `npm run build` in my next app

```
npm run build

> next-video-app@0.1.0 build
> next build

- info Loaded env from /Users/wes/Sites/mux/next-video-app/.env.local
- info Creating an optimized production build
- info Compiled successfully
- info Linting and checking validity of types ..(node:25973) [ESLINT_PERSONAL_CONFIG_SUPPRESS] DeprecationWarning: '~/.eslintrc.*' config files have been deprecated. Please remove it or add 'root:true' to the config files in your projects in order to avoid loading '~/.eslintrc.*' accidentally. (found in "../../../.eslintrc.json")
(Use `node --trace-deprecation ...` to show where the warning was created)
Failed to compile.

./app/page.tsx:4:26
Type error: Cannot find module '/videos/BBB-720p-1min.mp4' or its corresponding type declarations.

  2 | import styles from './page.module.css'
  3 | import Video from '@mux/next-video';
> 4 | import awesomeVideo from '/videos/BBB-720p-1min.mp4';
    |                          ^
  5 |
  6 | export default function Home() {
  7 |   return (
- info Linting and checking validity of types ...%
```

--- a/package.json
+++ b/package.json
@@ -15,7 +15,8 @@
       "import": "./dist/main.js",
       "require": "./dist/cjs/main.js",
       "default": "./dist/main.js"
-    }
+    },
+    "./video-types/*": "./video-types/*.d.ts"
   },
   "scripts": {
     "clean": "rm -rf dist",

